### PR TITLE
Gui debug mode

### PR DESCRIPTION
**Pull request by dalderliesten**
Changed the current GUI to be a special "debug" mode.  This mode can now be activated by starting the game with P, then clicking the 0 key.  It can be enabled/disabled using the same key.  Meant for debug purposes, and to not impose on the game status/existance.

Also fixed some naming and organization to ensure that the actual heads up display will be named just so.  The development for this will come soon.

### CODE REVIEW

File: src/main/java/nl/tudelft/pixelperfect/Game.java [hunk 1]
```diff
@@ -155,9 +156,9 @@ private void initNetwork() {
   @SuppressWarnings({ "checkstyle:methodlength", "PMD" })
   private void initInputs() {
     InputManager inputManager = getInputManager();
-    int[] keyTriggers = {KeyInput.KEY_W, KeyInput.KEY_S, KeyInput.KEY_A,
-        KeyInput.KEY_D, KeyInput.KEY_P};
-    String[] mappings = {"forward", "back", "left", "right", "start"};
+    int[] keyTriggers = { KeyInput.KEY_W, KeyInput.KEY_S, KeyInput.KEY_A, KeyInput.KEY_D,
+        KeyInput.KEY_P, KeyInput.KEY_0 };
+    String[] mappings = { "forward", "back", "left", "right", "start", "zero" };
```
Comment (wouterz) [inline src/main/java/nl/tudelft/pixelperfect/Game.java]: i suggest to rename zero to debug, as this is what it is trying to achieve

Comment (dalderliesten) [inline src/main/java/nl/tudelft/pixelperfect/Game.java]: Gotcha, push incoming.

**Comment by wouterz** [comment]
LGTM

**Comment by dalderliesten** [comment]
Has already been fixed, @jessetilro .

**Comment by jessetilro** [comment]
The static analysis reports are not clean.

**Comment by dmalarev** [comment]
code looks good to me

**Pull request merged**